tests/Behat/Resources/services.xml

```diff
@@ -3,6 +3,7 @@
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <parameters>
         <parameter key="sylius.behat.page.admin.channel.update.class">Tests\Sylius\InvoicingPlugin\Behat\Page\Admin\Channel\UpdatePage</parameter>
+        <parameter key="sylius_invoicing_plugin.knp_snappy.pdf.binary">/usr/local/bin/wkhtmltopdf</parameter>
```

Comment:
This parameter should be also configured for dev env, without that we would **force** user to configure it rather than providing some default configuration. We can configure it in `src/Resources/config/config.yml` under `parameters:` key. Then it can be mentioned in README that there is a default value set but it can be customized